src/client/websocket/WebSocketShard.js

```diff
@@ -59,7 +59,7 @@ class WebSocketShard extends EventEmitter {
      * @type {?string}
      * @private
      */
-    this.sessionID = null;
+    this.sessionID = sessionIDs;
```

Comment:
```suggestion
    this.sessionID = sessionID ?? null;
```
See above + should fallback to `null` if none was provided.